package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "build": "babel src --out-dir lib ",
     "build:watch": "babel src --watch --source-maps --out-dir lib",
-    "test:unit": "mocha --recursive ./src",
+    "test:unit": "mocha --recursive --require @babel/register ./src",
```

Comment:
Oppgradering til mocha 10 (som var påkrevd for å fjerne en kritisk vuln) knakk kjøring av testene. Har nok et eller annet å gjøre med at mocha har begynt å støtte ES Modules. Turte ikke å gå ned den stien, da jeg ikke rører denne pakken nok til å vite hvordan konsumenter blir påvirket. Fungerer fint å bygge testene on the fly, og vi har allerede `@babel/register` i repoet 🤷